cluster-autoscaler/cloudprovider/konvoy/konvoy.go

```diff
@@ -100,6 +100,12 @@ func (konvoy *KonvoyCloudProvider) Pricing() (cloudprovider.PricingModel, errors
 // NodeGroupForNode returns the node group for the given node.
 func (konvoy *KonvoyCloudProvider) NodeGroupForNode(node *apiv1.Node) (cloudprovider.NodeGroup, error) {
 	if _, found := node.ObjectMeta.Labels[KubernetesMasterNodeLabel]; found {
+		// Ignore node if it belongs to the master pool
+		return nil, nil
+	}
+
+	if _, found := node.ObjectMeta.Labels[nodeGroupLabel]; !found {
```

Comment:
Label name says `nodeGroupLabel` and comment says `autoscaling` label, which one is it?

Reply:
@mhrabovcin  it is the autoscaling label used to group nodes

Reply:
The node groups are identified based on this label which includes the word autoscaling ,perhaps I can be more explicit and add the label itself